superset/viz.py

```diff
@@ -144,15 +144,16 @@ def query_filters(self, is_having_filter=False):
         """Processes the filters for the query"""
         form_data = self.form_data
         # Building filters
+        flts = []
+        if is_having_filter and 'having_filters' in form_data:
+            flts = form_data['having_filters']
```

Comment:
it's not clear to me the difference between `flts` and `filters` unless i dig in and read more of the code. would be nice to make these var names more descriptive.